packages/astro/src/runtime/client/dev-toolbar/toolbar.ts

```diff
@@ -416,9 +416,9 @@ export class AstroDevToolbar extends HTMLElement {
 	}
 
 	getAppTemplate(app: DevToolbarApp) {
-		return `<button class="item" data-app-id="${app.id}">
+		return `<button class="item" data-app-id="${app.id}" aria-labelledby="app-name-${app.name}">
 				<div class="icon">${app.icon ? getAppIcon(app.icon) : '?'}<div class="notification"></div></div>
-				<span class="item-tooltip">${app.name}</span>
+				<span class="item-tooltip" id="app-name-${app.name}">${app.name}</span>
 			</button>`;
 	}
```

Comment:
Could you explain what’s intended here? I’m not 100% sure where this code is used, but in theory `aria-labelledby` should be unnecessary here as a `<button>` is already automatically labelled by its text content.

Reply:
I don't recall. I removed it :)